docs/build/hosted-lotus-node.md

```diff
@@ -0,0 +1,643 @@
+t154xvuihhicgluafwmohwzwmtmqp44pwwgewyvma  100 FIL          0
+```
+
+### Create a node.js project
+
+Let's also set up a Node.js project with necessary dependencies, we will write basic javascript functions to focus more on learning the code to interact with Filecoin Network. 
+
+1. **Create a new node.js project**
+
+   ```shell
+   mkdir build-with-lotus && cd build-with-lotus
+   npm init #Initialize an project
+   ```
+   
+2. **Install the fielcoin.js library**
+
+   ```shell
+   npm install filecoin.js
+   ```
+
+3. **Test a simple function call**
+
+   At this step, you are ready to write code to connect to your lotus node and interact with the lotus APIs. Let's write a simple script to query the lotus version to verify if everything works as expected.
+
+   First, let's create a `chainDataQuery.mjs` file in your project and add the following codes. We use `mjs` so we can use import statement in our code. 
+
+   ```javascript
+   //1. Import necessary packages
+   import { HttpJsonRpcConnector, LotusClient } from "filecoin.js";
+   
+   //2. Use the local node URL to create a connection to your lotus node
+   const localNodeUrl = "http://127.0.0.1:1234/rpc/v0";
+   const localConnector = new HttpJsonRpcConnector({ url: localNodeUrl });
+   
+   //3. lotusClient exposes all lotus APIs
+   const lotusClient = new LotusClient(localConnector);
+   const version = await lotusClient.common.version();
+   console.log(version);
+   ```
+
+   Run the following command in your project directory to test it.
+
+   ```shell with-output
+   ~/build-with-lotus $ node basicQuery.mjs
+   ```
+
+   ```shell
+   {
+     Version: '1.15.0+calibnet+git.0ac1bbc7a',
+     APIVersion: 66816,
+     BlockDelay: 30
+   }
+   ```
+
+   If everything is set up correctly, our code will be able to connect to the lotus node and request chain data back as shown above. And we are ready to explore more features that the lotus provides via API calls. 
+
+## Query chain data
+
+Blockchain data is the fundamental part of any blockchain network, it is essential that lotus provides APIs to query Filecoin chain data, including tipSets,  blocks, and messages, network status, storage deals and storage provider's details, etc. 
+
+As one of the API client libraries, filecion.js wrapped all the lotus APIs into simple method calls to provide  a complete library to interact with any lotus node. Let's look at some common packages of LotusClient so you can have a sense of what you can use. 
+
++ **LotusClient.common** - methods to get node info.
++ **LotusClient.chain** -  methods to interact with the blockchain.
++ **LotusClient.wallet** - methods to manage wallet on the lotus node.
++ **LotusClient.client** - methods to interact with the storage and retrieval markets as a client.
++ **LotusClient.paych** - methods to interact with and manage payment channels.
++ **LotusClient.state** - methods to query, inspect, and interact with chain state.
++ more ...
```

Comment:
If these are variables of methods, they should be between code tags:

```
`LotusClient.common`
```

https://docs.filecoin.io/community/contribute/grammar-formatting-and-style/#inline-code-tags